tests/app/test_credentialing.py

```diff
@@ -487,6 +487,22 @@ def test_credentialing_ends(helpers, seeder):
         assert res.json == {'description': f"registry EBVaw6pCqfMIiZGkA6qevzRUGsxTRuZXxl6YG1neeCGF not found",
                             'title': '404 Not Found'}
 
+        res = client.simulate_delete(f"/credentials/doesnotexist")
+        assert res.status_code == 404
+        assert res.json == {'description': f"credential for said doesnotexist not found.",
+                            'title': '404 Not Found'}
+
+        res = client.simulate_delete(f"/credentials/{saids[0]}")
+        assert res.status_code == 204
+
+        # Check db directly to make sure all indices are gone too (GET endpoints don't cover all indices)
```

Comment:
@iFergal Perhaps an assertion that you get HTTP 404 (and not for example, a 500), after the deletion could be added here? Just to ensure that the databases are not left in an incomplete state that causes errors.

Reply:
good point, will update